README.md

```diff
@@ -0,0 +1,70 @@
+## About
+
+This is a minimal JavaScript reader for the [TRX tractography file format](https://github.com/tee-ar-ex/trx-spec/blob/master/specifications.md). Previously, most tractography tools used their own proprietary [format](https://www.nitrc.org/plugins/mwiki/index.php/surfice:MainPage#Supported_Formats) such as BFLOAT, niml.tract, PDB, [TCK](https://mrtrix.readthedocs.io/en/latest/getting_started/image_data.html#tracks-file-format-tck), [TRK](http://trackvis.org/docs/?subsect=fileformat) and VTK. The TRX format was developed from community [discussions](https://github.com/nipy/nibabel/issues/942) to address the [limitations of the existing formats](https://docs.google.com/document/d/1GOOlG42rB7dlJizu2RfaF5XNj_pIaVl_6rtBSUhsgbE/edit#heading=h.c6igqydj1hrf) and the needs of the users. This reader supports all the features of TRX including data-per-vertex, data-per-streamline and groups.
+
+## Live Demo
+
+[NiiVue provides a WebGL live demo of this code](https://niivue.github.io/niivue/features/tracts.html). This can be tested on any device (computer, laptop, phone). A sample TRX file is loaded by default, but users can drag and drop new streamlines in the niml.tract, TCX, TRK, TRX, and VTK formats.
+
+## Node.JS Command Line Demo
+
+The module `streamlineIO.mjs` provides JavaScript reading for streamlines in the TCX, TRK, TRX, and VTK formats. The included file `bench.mjs` demonstrates these capabilities by loading a streamline 11 times and reporting details regarding file size, loading time, and streamline properties.
+
+To run the demo:
+
+```
+$ git clone https://github.com/tee-ar-ex/trx-javascript
+$ cd trx-javascript
+$ npm install gl-matrix fflate fzstd
```

Comment:
Is it possible to write what is the correct version of Node?
I used `sudo apt` and the version was so old the code kept crashing on the first line.
When I installed it directly from a `.tar.gz` from their website it worked immediately. 

`Install the appropriate version of the most recent [NodeJS](https://nodejs.org/en/download/).`

Reply:
@neurolabusc : do you want to address this question in this PR, or should we merge and add this as an issue for future work?